### PR TITLE
Removed not necessary NIB MSDN link

### PR DESCRIPTION
**Pull request by pkulikov**
This PR removes/replaces the following NIB link
`[NIB: How to: Handle Exceptions Thrown by Tasks](https://msdn.microsoft.com/library/d6c47ec8-9de9-4880-beb3-ff19ae51565d)`

from the docs.

It's replaced by 
https://docs.microsoft.com/en-us/dotnet/standard/parallel-programming/exception-handling-task-parallel-library

that contains the content of the MSDN link and much more.

Contributes to #1787

Also fixed couple of typos in the dataflow topic.

### CODE REVIEW

File: docs/standard/parallel-programming/dataflow-task-parallel-library.md [hunk 1]
```diff
@@ -66,14 +66,14 @@ ms.author: "ronpet"
...
+ You can also use properties such as <xref:System.Threading.Tasks.Task.IsCanceled%2A> in the body of the continuation task to determine additional information about the completion status of a dataflow block. For more information about continuation tasks and how they relate to cancellation and error handling, see [Chaining Tasks by Using Continuation Tasks](../../../docs/standard/parallel-programming/chaining-tasks-by-using-continuation-tasks.md), [Task Cancellation](../../../docs/standard/parallel-programming/task-cancellation.md), [Exception Handling](../../../docs/standard/parallel-programming/exception-handling-task-parallel-library.md), and [Exception Handling](../../../docs/standard/parallel-programming/exception-handling-task-parallel-library.md).
```
Comment (rpetrusha) [inline docs/standard/parallel-programming/dataflow-task-parallel-library.md]: The Exception Handling article is duplicated here.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for removing the unused link from these articles, @pkulikov. There's one duplicated reference to the Exception Handling article. Once that's removed, this is ready to merge.

File: docs/standard/parallel-programming/dataflow-task-parallel-library.md (hunk 1 above)
Comment (pkulikov) [reply docs/standard/parallel-programming/dataflow-task-parallel-library.md]: @rpetrusha thanks! Fixed it.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional change, @pkulikov. I'll merge your PR now.

**Pull request merged**